spec/features/search/show_spec.rb

```diff
@@ -20,15 +20,18 @@
         expect(current_path).to eq results_path
         expect(page).to have_link('Fight Club')
         expect(page).to have_content("Vote Average: 8.4")
-        expect(page).to have_css(".movie", count: 22)
+        expect(page).to have_css(".movie", count: 20)
       end
     end
     it 'when I click top 40, I see the top 40 search results' do
       VCR.use_cassette('movies_top_40') do
+        movie_details = SearchFacade.top_rated(1)
+        movie_detail = movie_details.first
+    
         click_on 'Discover Top 40'
-        expect(page).to have_link('Fight Club')
-        expect(page).to have_content('Vote Average: 8.4')
-        expect(page).to have_css('.movie', count: 40)
+        expect(page).to have_link(movie_detail.title)
+        expect(page).to have_content("Vote Average: #{movie_detail.vote_average}")
+        expect(page).to have_css('.movie', count: 20)
```

Comment:
I think this test will need to revert back to 40 once you call consolidate results in a file comment I made prior. WE are expecting there to be 40 results and there should be based on the original card.